### ISSUE TITLE
Training loss nan

### ISSUE THREAD

**Issue by IceIce1ce** [ngbrjyj/A2LC#3]
Hi author, thanks for your great code. When I run the training script for the VOC2012 dataset (bash DeepLabV3Plus_Pytorch/run_pascal.sh), the training loss is nan. My pytorch version is 2.1.0. Does this problem only occur with the new version of PyTorch? (The pytorch version 1.13.1 can work well).

**Comment by ngbrjyj** [comment]
Hello,

I confirm that the training runs successfully in my environment.

For reference, my environment is as follows:
  OS: Ubuntu 20.04
  CUDA: 11.6
  cuDNN: 8.4.1
  Python: 3.8
  PyTorch: 1.13.1

I hope this helps clarify the issue.

**Comment by IceIce1ce** [comment]
Hi author, thanks for your reply. I can confirm that the old pytorch version (1.13.1) can work well. However, when I try the new pytorch version (>= 2.0), the training loss will become nan. Anyway, thanks for your help.
